docs/libcurl/curl_version_info.3

```diff
@@ -216,6 +223,89 @@ backends). (Added in 7.21.4)
+names of the features that libcurl supports. The array is terminated by a NULL
+entry. Currently defined names are:
+.RS
+.IP alt-svc
+HTTP Alt-Svc parsing and the associated options.
+.IP AsynchDNS
+libcurl was built with support for asynchronous name lookups, which allows
+more exact timeouts (even on Windows) and less blocking when using the multi
+interface.
+.IP brotli
+supports HTTP Brotli content encoding using libbrotlidec.
+.IP Debug
+libcurl was built with debug capabilities.
+.IP gsasl
+libcurl was built with libgsasl and thus with some extra SCRAM-SHA
+authentication methods.
+.IP GSS-API
+libcurl was built with support for GSS-API. This makes libcurl use provided
+functions for Kerberos and SPNEGO authentication. It also allows libcurl
+to use the current user credentials without the app having to pass them on.
+.IP HSTS
+libcurl was built with support for HSTS (HTTP Strict Transport Security).
+.IP HTTP2
+libcurl was built with support for HTTP2.
+.IP HTTP3
+HTTP/3 and QUIC support are built-in.
+.IP HTTPS-proxy
+libcurl was built with support for HTTPS-proxy.
+.IP IDN
+libcurl was built with support for IDNA, domain names with international
+letters.
+.IP IPv6
+supports IPv6.
+.IP Kerberos
+supports Kerberos V5 authentication for FTP, IMAP, LDAP, POP3, SMTP and
+SOCKSv5 proxy.
+.IP Largefile
+libcurl was built with support for large files.
+.IP libz
+supports HTTP deflate using libz.
+.IP MultiSSL
+libcurl was built with multiple SSL backends. For details, see
+\fIcurl_global_sslset(3)\fP.
+.IP NTLM
+supports NTLM authentication mechanism.
+.IP NTLM_WB
+libcurl was built with support for NTLM delegation to a winbind helper.
+.IP PSL
+libcurl was built with support for Mozilla's Public Suffix List. This makes
+libcurl ignore cookies with a domain that is on the list.
+.IP SPNEGO
+libcurl was built with support for SPNEGO authentication (Simple and Protected
+GSS-API Negotiation Mechanism, defined in RFC 2478).
+.IP SSL
+supports SSL/TLS.
+.IP SSPI
+libcurl was built with support for SSPI. This is only available on Windows and
+makes libcurl use Windows-provided functions for Kerberos, NTLM, SPNEGO and
+Digest authentication. It also allows libcurl to use the current user
+credentials without the app having to pass them on.
+.IP threadsafe
+libcurl was built with thread-safety support (Atomic or SRWLOCK) to protect
+curl initialization. See \fIlibcurl-thread(3)\fP.
+.IP TLS-SRP
+libcurl was built with support for TLS-SRP (in one or more of the built-in TLS
+backends).
+.IP TrackMemory
+libcurl was built with memory tracking debug capabilities. This is mainly of
+interest for libcurl hackers.
+.IP Unicode
+libcurl was built with Unicode support on Windows. This makes non-ASCII
+characters work in filenames and options passed to libcurl.
+.IP UnixSockets
+libcurl was built with support for Unix domain sockets.
+.IP zstd
+supports HTTP zstd content encoding using zstd library.
+.RE
```

Comment:
This is much better, but now you've duplicated the documentation for each feature, and they're not duplicated identically. I suggest we instead list each feature with both name and bitmask define name, and have *one* explanation for it.

Reply:
Can I drop descriptions of `CURL_VERSION_CONV` and `CURL_VERSION_KERBEROS4`? They're obsolete and have no associated name. Reason: I want to label descriptions by names and these are not named! Or do I associate a dummy name that can never occur :-( ?

Reply:
> each feature with both name and bitmask define name, and have one explanation for it.

Done.
(I wish man had a built-in `.TABLE` macro!)